oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RunningRule.java

```diff
@@ -383,6 +396,17 @@ private boolean isMatch() {
             return isMatch == 1;
         }
 
+        public boolean checkOutOfDate() {
```

Comment:
```suggestion
        public boolean isExpired() {
```

Please use `expired` to replace `out-of-date`. Because `out-of-date` is more about idea, such as old docs and design.